client/components/country-dropdown/index.js

```diff
@@ -17,7 +17,6 @@ const CountryDropdown = ( props ) => {
 CountryDropdown.propTypes = {
 	id: PropTypes.string.isRequired,
 	countriesData: PropTypes.object.isRequired,
-	valuesMap: PropTypes.object.isRequired,
```

Comment:
Because the CountryDropdown component doesn't receive the `valuesMap` property, instead it gets the values from the list of `storeOptions.countriesData`. This PropType definition was mistakingly kept from a previous refactor.